simulation_ws/src/robot_fleet/scripts/client_mqtt.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
```

Comment:
@kapilPython  client_mqtt, and gazebo_model_mover_mqtt seem to have a lot of common code with the original files. The creation of different files to just change the model of communication will require keeping the two files in sync for any logic change.
Can you consider setting it up so that we dont have to duplicate the code?

Reply:
@konduri yeah keeping the files same by having two alternatives for communication seems logical. Would update the commit soon after testing suggested setup.